drivers/quota/projectquota.go

```diff
@@ -428,7 +428,11 @@ func makeBackingFsDev(home string) (string, error) {
 	backingFsBlockDevTmp := backingFsBlockDev + ".tmp"
 	// Re-create just in case someone copied the home directory over to a new device
 	if err := unix.Mknod(backingFsBlockDevTmp, unix.S_IFBLK|0o600, int(stat.Dev)); err != nil {
-		return "", fmt.Errorf("failed to mknod %s: %w", backingFsBlockDevTmp, err)
+		// On errors, try again after unlinking any potential leftover.
```

Comment:
Wouldn't it be safer to only try this on EEXIST?

Reply:
Yes only do this if error.Is(err, EEXIST)

Reply:
changed to be retried only on EEXIST